source/plugins.rst

```diff
@@ -204,15 +204,17 @@ For an importer, synfig will pass the file selected in the open dialog as first
 Once the script is completed, synfig will load that second file, so the plugin script should populate it appropriately.
 
 
-= Tutorial =
+Tutorial
+=====
```

Comment:
Thank you! Just one note: please make sure that underlining is at least as long as the text. ^__^
Details - https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections